### ISSUE TITLE
APKs missing from v3.26 release assets on GitHub 

### ISSUE THREAD

**Issue by w9925** [nextcloud/android#12020]
Please Upload the 3.26 apk to the release.

**Comment by joshtrichards** [comment]
Hi @w9925 - Where, specifically, are you referring to?

**Comment by joshtrichards** [comment]
Agreed. Usually the APK assets are included but they're not there on GitHub for 3.26.0 for some reason:

https://github.com/nextcloud/android/releases/tag/stable-3.26.0

Cc: @tobiasKaminsky

**Comment by joshtrichards** [comment]
Looking into it (both items).
